_opensearch/snapshots/snapshot-restore.md

```diff
@@ -343,6 +343,7 @@ Setting | Description
 `rename_replacement` | If you want to rename indices as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching index name, `$1` to include the content of the first capture group, etc.
 `index_settings` | If you want to change index settings on restore, specify them here.
 `ignore_index_settings` | Rather than explicitly specifying new settings with `index_settings`, you can ignore certain index settings in the snapshot and use the cluster defaults on restore.
+`storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the [search role]({{site.url}}{{site.baseurl}}/security-plugin/access-control/users-roles/) in order to restore a snapshot using the type `remote_snapshot`. <br /><br > Defaults to `local`.
```

Comment:
Roles [in the security plugin](https://opensearch.org/docs/latest/security-plugin/access-control/users-roles/) are a different concept. The only existing documentation I've found for configuring the node roles is in the [ML plugin](https://opensearch.org/docs/latest/ml-commons-plugin/index/#ml-node). I haven't found any documentation around the built-in node roles (cluster_manager, data, ingest, etc) that would be appropriate to link here.